src/main/scala/com/gu/support/tstash/TstashAppender.scala

```diff
@@ -64,9 +64,15 @@ class TstashAppender extends UnsynchronizedAppenderBase[ILoggingEvent] {
     }
   }
 
-  def sendHTMLFile(tstashURL: String): Unit = {
+  def sendHTMLFile(tstashURL: String, testId: String): Unit = {
     val tstashReportHtml = s"<html><head><meta <meta http-equiv='refresh' content='0; url=$tstashURL' /></head><body><a href='$tstashURL'>Test report</a></body></html>"
-    Files.write(Paths.get("TstashReport.html"), tstashReportHtml.getBytes(StandardCharsets.UTF_8))
+    if (testId.startsWith("Some")) {
```

Comment:
there's probably a better way to do that, just do a getOrElse('Single') in the logger and then just either use what's given or Single otherwise